src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.Operations
 {
-    internal sealed class OperationCloner : OperationVisitor<object, IOperation>
+    internal class OperationCloner : OperationVisitor<object, IOperation>
```

Comment:
Is it possible to make it abstract, then?

Reply:
> Is it possible to make it abstract, then?

No.

---
In reply to: [162165962](https://github.com/dotnet/roslyn/pull/24263#discussion_r162165962) [](ancestors = 162165962)